pkg/preference/preference.go

```diff
@@ -367,6 +380,16 @@ func (c *PreferenceInfo) SetConfiguration(parameter string, value string) error
 			}
 			c.OdoSettings.PushTimeout = &typedval
 
+		case "registrycachetime":
```

Comment:
please add a test case for this in the test function of SetConfiguration.

Reply:
added